ModPatches/Medieval Overhaul Barbarians/ThingDefs_Misc/Shields/BarbarianBannerShields.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<Operation Class="PatchOperationRemove">
```

Comment:
Copy these over to the format I used for #3328--you should be able to just copy-paste.